backend/main.py

```diff
@@ -114,6 +116,15 @@ def GetGun():
   return jsonify(GetFirebase().get('/guns', gun))
 
 
+@app.route('/cronNotification', methods=['GET'])
+def CronNotification():
+  cron_key = 'X-Appengine-Cron'
+  if cron_key not in request.headers or not request.headers[cron_key]:
+    return 'Unauthorized', 403
+  notifications.ExecuteNotifications(None, GetFirebase())
```

Comment:
I thought about it, but I don't think cron actually cares, so I only return something if someone other than cron tries to hit that endpoint.